### ISSUE TITLE
Browse Windows expand to fit all content

### ISSUE THREAD

**Issue by paulhoule** [paulhoule/tentacruel#10]
This is part of #2.

Right now the scrollbars appear,  but they are not active because the window is full with results;
it expands to fit all of the results,  even if the window hangs far off the bottom of the screen.

**Comment by paulhoule** [comment]
Ok,  to set the size of a window we have to use the "geometry" method.

When I lock this down at the beginning I often get a window that is too big.  I think I need something to lock down the geometry to a maximum after (or as we) load the window.

**Comment by paulhoule** [comment]
So I struggled just a little bit to figure out when exactly to do the geometry change and how to get a reference to the window there.

For instance if I tried to do this in the fill_source() method I don't know if the geometry change would be visible then or after the UI gets updated.

Then I looked at it and saw that the update_region method gets called when the geometry of the frame is changed,  and probably the size of the window updates with that.  Let's check.

Actually the geometry that I get for that window is always the same,  at least in size:

```
723x51+104+104
```

or

```
723x51+130+130
```

which makes me wonder if we are seeing the wires get crossed in the static scope or something.
Note that the 723x51 wide but not very tall,  it is about what I'd expect before the window expanded.  That may not be all that strange,  however,  it could be that the <Configure> hook gets called for the frame and then everything works outward from there.  We certainly could lock down the geometry of the window from here but then it might not be reflective of all the chrome...
